### DISCUSSION TITLE
CNBingo Thermostat zigbee

### DISCUSSION THREAD

**Discussion by Olih1985** [Koenkk/zigbee2mqtt#21813] [Devices]
I'm new to this topic ans try to connect a CNBingo Thermostat in Homeassistant via zigbee2mqtt.
My problem ist that the device is connecting, but not recognised- unknown.
I can't find out, how I can add the device manually.
I'm really new and the help pages are not clear. I can't even find the yaml files needed on the homeassistant (using homeassitant green). So I would need an explanation ation for noobs...

**Comment by lux73** [comment]
https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

i must never use this - but it's worth a try for your Device 😉
